File: apps/src/chat/frontend/js/packets/websocket/TotalUserPacket.js
export default class TotalUserPacket {

    constructor(online_users){
        this.online_users = online_users;
    }
    
    toData(){
        return {
            type: "total_user",
            online_users: this.online_users
        }
    }

    static fromData(data){
        return new TotalUserPacket(
            data["online_users"]
        );
    }

    getOnlineUsers(){
        return this.online_users;
    }
}

File: apps/src/chat/backend/packets/websocket/total_user_packet.py
from ..packet import Packet

class TotalUserPacket(Packet):

    def __init__(self, online_users: int):
        self._online_users = online_users

    def to_data(self):
        return {
            "type": "total_user",
            "online_users": self._online_users
        }

    @staticmethod
    def from_data(data):
        return TotalUserPacket(
            data["online_users"]
        )